### ISSUE TITLE
Personal / Tech blog switch

### ISSUE THREAD

**Issue by akaralar** [rushiagr/rushiagr.github.com#2]
I'm trying to open my first blog and I saw that you did a really nice change between personal and tech blog. How did you go about doing this in Hugo? Do you create 2 separate blogs and modify the generated site files? Is there an automatic way to do this? I was generally curious about how you went about doing that. It looks really nice, congrats! :)

**Comment by rushiagr** [comment]
Thanks Ahmet, glad you find this useful. Yes, I just copied the original repository (`npf`) to create another one (`npfpersonal`), and changed the 'baseurl' in config.toml of personal blog to:

```
baseurl = "http://www.rushiagr.com/personal"
```

From then on, I created a small script to copy over the generated content from personal blog to generated content of the primary blog. The script can be found here: https://github.com/rushiagr/myutils/blob/master/otherfiles/publish.sh

This what the quickest solution I can come up at that time. It might be possible to do such a thing directly in Hugo, but I thought my way would be faster and sure to work ;)
